src/player.cpp

```diff
@@ -6445,6 +6447,9 @@ void player::hardcoded_effects(effect &it)
             }
         }
     } else if( id == effect_bleed ) {
+        // Prolonged haemorrhage is a significant risk for developing anaemia
+        vitamin_mod( vitamin_iron, rng( -1, -4 ) );
```

Comment:
> how significant that 180 (average 75) points of iron deficiency would be

Couldn't cause anemia on it's own but if they player was already low on iron and the effect was repeated could occur